pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTClassType.java

```diff
@@ -125,6 +125,22 @@ public ASTTypeArguments getTypeArguments() {
     }
 
 
+    /**
+     * Return the package qualifier, if this is a fully qualified name.
+     * Note that this will only be the case if we could resolve the
+     * qualifier to a package name during disambiguation. In other words,
+     * if the auxclasspath is not complete, and the qualifier could not
+     * be disambiguated, this method will return null (and an AmbiguousName
+     * will stay in the tree).
+     */
```

Comment:
we should probably call out that inner classes don't get a package qualifier either, regardless of auxclasspath…